SUPPORT.md

```diff
@@ -0,0 +1,22 @@
+<!-- https://help.github.com/en/github/building-a-strong-community/adding-support-resources-to-your-project -->
+
+# Support
+
+Having trouble? Please consult these official resources in order of appearance.
+
+## Resources
+
+**[open.inf.is][docs-site] on the Web:** The OpenINF website. It has docs and
+tutorial resources.
+
+**[`#openinf`][matrix-channel] on matrix.org:** The OpenINF Matrix support channel
+on the _matrix.org_ network. It’s a synchronous chat channel for lightweight
+communication. Good uses of it are for asking questions, casual conversation,
+collaboration, and quick exchanges.
+
+**[@openinf][twitter-account] on Twitter:** The OpenINF support account on
+Twitter.
```

Comment:
```suggestion
Twitter. Be sure to follow the [Twitter Rules] and our [Code of Conduct].
```

Reply:
```html
<abbr title="Direct Messages">DMs</abbr>
```

> <abbr title="Direct Messages">DMs</abbr>

but github markdown render engine non-supported.

Reply:
That's alright, these will end up being processed by another Markdown process (not necessarily the one GitHub for GitHub-flavored Markdown).